src/puppetlabs/puppetdb/mq.clj

```diff
@@ -243,7 +243,7 @@
 
 (defn send-message!
   "Sends message with attributes via a new session (sessions are
-  single-threaded)."
+   single-threaded)."
```

Comment:
Hmm, do we have a convention here -- seems like our code has both, but looks like by default clojure-mode (at least) uses the first style (via M-q).